jinja/plugins.html

```diff
@@ -31,22 +31,33 @@ <h4 class="card-header juicy-peach-gradient">default.qubit</h4>
                 </a>
             </div>
             <div class="card plugin-card">
-                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.plugins.default_gaussian.DefaultGaussian.html">
-                    <h4 class="card-header juicy-peach-gradient">default.gaussian</h4>
+                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.plugins.default_qubit_tf.DefaultQubitTF.html">
+                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf</h4>
                     <div class="card-body">
                         <p class="card-text">
-                            A simple quantum photonic simulator written in Python.
-                            <br><br>A good choice for optimizating photonic systems.
+                            A state-vector qubit simulator written using TensorFlow and supporting classical backpropagation. <br><br>A good choice when the number of parameters to be optimized is large.
                         </p>
                     </div>
                 </a>
             </div>
             <div class="card plugin-card">
-                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.plugins.default_qubit_tf.DefaultQubitTF.html">
-                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf</h4>
+                <a href="https://pennylane-lightning.readthedocs.io">
+                    <h4 class="card-header juicy-peach-gradient">lightning.qubit (beta)</h4>
                     <div class="card-body">
                         <p class="card-text">
-                            A state-vector qubit simulator written using TensorFlow and supporting classical backpropagation. <br><br>A good choice when the number of parameters to be optimized is large.
+                            A state-vector qubit simulator written with a C++ backend.
+                            <br><br>A good choice if you want to run faster than default.qubit.

```

Comment:
```suggestion
                            A fast state-vector qubit simulator written with a C++ backend.
                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters.
```
@josh146, how about this? The "A good choice" is same as for default.qubit, but we say it's fast in the sentence above.